_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,131 @@
+
+### Use the time filter
+
+1. In the Dashboards console, select **Discover** and confirm index pattern being used is `opensearch_dashboards_sample_data_ecommerce`.
+2. Select calendar icon to change time field. The default is **Last 15 minutes**.
+3. Change the time field to **Last 7 days** and select **Refresh**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-9.png" alt="Time filter user interface">
+
+4. To set the start and end times, select the bar next to the time filter. In the popup, select **Absolute**, **Relative**, or **Now**, then specify the required options.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-10.png" width=300 alt="Start and end times user interface">
+
+### Select a time range from the histogram
+
+To select a time range for the histogram, you can choose one of the following:
+
+* Select the bar that represents the time range you want to zoom in on.
+* Select the bar and drag to view a specific time range. You must start the selection with the cursor over the background of the chart—the cursor changes to a plus sign when you hover over a valid start point.
+* Select the dropdown, then select an interval.
+
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-11.jpg" alt="Histogram user interface">
+
+## Create visualizations
+
+Follow these steps to learn how to connect your visualizations in a single dashboard.
+
+1. In the Dashboards console, select **Visualize** > **Create visualization**.
+2. Select the visualization type. For this tutorial, select **Line**.
+3. Choose a source. For this tutorial, select the index pattern `opensearch_dashboards_sample_data_ecommerce`.
+4. Under **Buckets**, select **Add > X-axis**.
+5. In the **Aggregation** field, select **Date Histogram** and then **Update**.
+6. Select **Save** and add file name. This tutorial uses preconfigured visualizations, so you won’t be able to save your visualization.
+
+## Connect visualizations in a single dashboard
+
+1. In the Dashboards console, select **Dashboard > Create dashboard**. 
+2. Select **Add an existing** and then select the data you want to add.
+3. Select **Save** and add dashboard name in the **Title field**. This tutorial uses preconfigured dashboards, so you won’t be able to save your dashboard.
+4. Click into dashboard white space left of **Add panels** to view the visualizations in a single dashboard.
+
+Your dashboard might look like this:
+
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-12.jpg" width=500 alt="Example dashboard using data visualizations from many data sources">
+
+You have explored the data sources experimental feature. We look forward to your feedback on how we can improve this feature ahead of its release for production use.
+
+## Limitations
+
+The following limitations apply to the 2.4 release of this experimental feature:
+
+* The multiple data sources currently only works with index pattern based visualizations
```

Comment:
Again, "multiple data sources ... works with..." rubs me the wrong way.

Reply:
Revised.